src/mozmlops/templates/README.md

```diff
@@ -0,0 +1,63 @@
+# Training and Deploying Models in The Cloud
```

Comment:
Please add a reference to templates (and kindly point to this doc!) in the top level README.

Reply:
Done!